etc/templates/web/nginx-location.conf.template

```diff
@@ -0,0 +1,30 @@
+#sendfile on;
```

Comment:
Perhaps either remove these or extend this comment block to say that such properties should be set by the sysadmin. Perhaps include a full working example?